appcues/src/main/java/com/appcues/debugger/ui/AppcuesSearchView.kt

```diff
@@ -75,7 +73,10 @@ internal fun AppcuesSearchView(
             .border(1.dp, border.value, RoundedCornerShape(cornerDp.value))
             .background(LocalAppcuesTheme.current.background)
             .height(height)
-            .clickable(interactionSource = MutableInteractionSource(), indication = null) { focusRequester.requestFocus() },
+            .clickable(
+                interactionSource = remember { MutableInteractionSource() },
```

Comment:
fixing [warning](https://googlesamples.github.io/android-custom-lint-rules/checks/UnrememberedMutableInteractionSource.md.html) about MutableInteractionSource created outside of remember.